extensions/amp-izlesene/0.1/amp-izlesene.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ import {addParamsToUrl} from '../../../src/url';
+ import {getDataParamsFromAttributes} from '../../../src/dom';
+ import {isLayoutSizeDefined} from '../../../src/layout';
+ import {dev, user} from '../../../src/log';
+
+ class AmpIzlesene extends AMP.BaseElement {
+  constructor(element) {
+    super(element);
+    /** @private {?string}  */
+    this.videoid_ = null;
+    /** @private {?Element} */
+    this.iframe_ = null;
+    /** @private {?string} */
+    this.videoIframeSrc_ = null;
+  }
+
+  /**
+   * @param {boolean=} opt_onLayout
+   * @override
+   */
+  preconnectCallback(opt_onLayout) {
+    this.preconnect.preload(this.getVideoIframeSrc_());
```

Comment:
`preload` is too heavy. Please just do `this.preconnect.url(this.getVideoIframeSrc_());` to preconnect.